packages/commons-test/test/jwt.unit.test.ts

```diff
@@ -172,12 +172,12 @@ describe("JWT tests", () => {
       ]);
       const token = getMockSignedToken({
         ...mockToken,
-        sub: undefined,
+        iss: undefined,
         jti: undefined,
       });
 
       expect(() => readAuthDataFromJwtToken(token)).toThrowError(
-        invalidClaim(`Validation error: Required at "jti"; Required at "sub"`)
+        invalidClaim(`Validation error: Required at "iss"; Required at "jti"`)
```

Comment:
When `mockToken` is set to `UiToken` it lacked the `sub` field and the test failed. To address this, I replaced it with a common field across `AuthToken` schemas.